lite/.github/workflows/build.yml

```diff
@@ -0,0 +1,212 @@
+name: Build
```

Comment:
These workflows should be hoisted to the top-level so they can actually run.

Reply:
The work is being done in separate PRs against the [merge-xeus-python-kernel](https://github.com/jupyter-xeus/xeus-python/tree/merge-xeus-python-kernel) branch